go.mod

```diff
@@ -32,4 +33,5 @@ require (
 	k8s.io/client-go v0.17.2
 	k8s.io/utils v0.0.0-20200414100711-2df71ebbae66 // indirect
 	sigs.k8s.io/controller-runtime v0.4.0
+	sigs.k8s.io/controller-tools v0.2.4 // indirect
```

Comment:
Hmm, is this required?
I think you can remove this line and run `go mod tidy`

Reply:
I did not write that, I think Visual Studio Code put it automatically.

Reply:
Yeah, it's indirect - can you try removing that line and running `go mod tidy` and see if it comes back?

Reply:
It worked, thanks.

Reply:
Actually, it keeps getting added. I will delete it before pushing.